### ISSUE TITLE
All the apps don't accept HTTPS Adguard certificate when it's moved to system

### ISSUE THREAD

**Issue by Juloboy** [AdguardTeam/AdguardForAndroid#4095]
### Prerequisites

Please answer the following questions for yourself before submitting an issue. **YOU MAY DELETE THE PREREQUISITES SECTION.**

- [Yes] I am running the latest version
- [Yes] I checked the documentation and found no answer
- [Yes] I checked to make sure that this issue has not already been filed

### Issue Details

 

- **AdGuard version:**
  -  3.6.6
- **Filtering mode:**
  - HTTP Proxy Auto
- **Device:**
  - POCO X3 NFC 
- **Operating system and version:**
  - Arrow OS Android 11 Custom ROM (January Security Patch)
- **Root access:**
  - Yes

### Actual Behavior

Adguard is sending notifications that ALL apps doesn't accept it's HTTPS certificate. Certificate is moved to system via Move certificates Magisk module. There's no such issue when Adguard HTTPS certificate is located in user's certificates.

### Screenshots

![Screenshot_20220122-122618](https://user-images.githubusercontent.com/59500988/150660470-bc9a38b8-661c-4f88-b464-af5c79126d44.png)
![Screenshot_20220122-124724](https://user-images.githubusercontent.com/59500988/150660471-359f2369-f799-4c7a-a0e2-4724ad88de28.png)
![Screenshot_20220122-122044](https://user-images.githubusercontent.com/59500988/150660472-c27f28c4-7f66-498c-adab-4adcf80a4812.png)
![Screenshot_20220122-124637](https://user-images.githubusercontent.com/59500988/150660474-da5d5712-07bc-42ab-b412-e5a90b95be8a.png)

<details><summary>Screenshot:</summary>

</details>

### Additional Information

 I didn't have such issue for years of usage Adguard. Didn't have this being on the same ROM with December Security Patch. I got this issue after updating to the ROM with January Security Patch, maybe it can be the reason.

**Comment by Chinaski1** [comment]
Hello there!

It seems that AdGuard cannot find the security certificate in the system storage. 
You may have saved them in a different directory.
Try reinstalling the certificate through Magisk first.

**Comment by Juloboy** [comment]
> Hello there!
> 
> It seems that AdGuard cannot find the security certificate in the system storage. You may have saved them in a different directory. Try reinstalling the certificate through Magisk first.

I reinstalled the app and the certificates several times, it didn't help. Can the log be helpful for you?

**Comment by lancelot-moon** [comment]
@Juloboy 
If you correctly moved AG certificate into system store and then found many apps don't accept AG HTTPS filtering, this is normal.

Simply, click "ignore for app".
By default, many apps already don't accept AG certificate.
That's why you saw those warnings.

**Comment by whalehub** [comment]
@Juloboy I recently started experiencing the same issue after years of using AdGuard's HTTPS-filtering feature successfully. I ended up creating my own Magisk module to fix it. Unlike the other Magisk module, it doesn't copy certificates from the user store to the system store, it copies them from the file system to the system store, meaning that they don't need to be installed in the user store to begin with. Maybe it will fix the issue for you as well: https://github.com/whalehub/custom-certificate-authorities
